spec/asyncapi.md

```diff
@@ -139,6 +139,21 @@ This is applicable for `$ref` fields in the specification as follows from the [J
 
 By convention, the AsyncAPI Specification (A2S) file is named `asyncapi.json` or `asyncapi.yaml`.
 
```

Comment:
The spec should define "traits" before explaining their behavior. My suggestion from the PR conversation:
```
### <a name="traits"></a>Traits

Traits are pieces of information that will be merged into the referencing object. Using traits reduces the amount of repetitive code within an AsyncAPI document.
```

Reply:
I've added this now to the PR just above the ### Trait Inheritance Section